### PR TITLE
Set user sessions to DB in config.php

### PR DESCRIPTION
**Pull request by roperto**

### CODE REVIEW

File: scripts/install_moodle.sh [hunk 1]
```diff
@@ -883,6 +883,9 @@ EOF
    #else # mssql. TODO It's missed earlier! Complete this!
    fi
 
+   # setting user sessions to DB
+   sed -i "23 a \$CFG->session_handler_class = '\\\core\\session\\\database';" /moodle/html/moodle/config.php
```
Comment (hosungsmsft) [inline scripts/install_moodle.sh]: We did try this as well, but it didn't give any noticeable perf improvement. Still we can have this, but I'd like this to be grouped together with code where the `/moodle/html/moodle/config.php` is updated, like line 860. I think the 3 PRs could have been just one.

**Comment by superzoelu** [comment]
@hosungsmsft - sorry for the confusion, this change request is to set user session handler to database when the template is not using redis for user sessions. I have updated the code to reflect this change.

**Comment by ummadisudhakar** [comment]
We can close this one as this does not have any significance proof that it will improve the performance

**Comment by UmakanthOS** [comment]
Closing this per above testing and comments.

**Pull request closed without merging**